### DISCUSSION TITLE
以前meta内核大小也近30m吗

### DISCUSSION THREAD

**Discussion by igmkgu** [MetaCubeX/mihomo#1562] [General]
把openclash内核换成mihomo-linux-arm64发现路由器就剩10m内存了，我记得之前用meta内核好像没这么大，我怎么才能把内核大小缩一下，还是说我下错版本了？

**Comment by fejelaqbhbv** [comment]
openclash提供的内核是作者自己二次压缩过的
看版本号都不一样，多了一个g

**Comment by refined-fish** [comment]
如L1所言，你需要的话就去openclash那里下载就好了，那个经过了压缩

**Comment by Xholam** [comment]
不知道 mihomo 内核是否开源的？见到体积这么大有点不放心
